extensions/amp-lightbox-viewer/0.1/service/lightbox-manager-impl.js

```diff
@@ -120,20 +135,57 @@ export class LightboxManager {
    * @private
    */
   processLightboxElement_(element) {
-    if (element.tagName == 'AMP-CAROUSEL') {
-      // TODO: special case carousel
+    if (element.tagName.toLowerCase() == CAROUSEL_TAG) {
+      let lightboxGroupId = element.getAttribute('lightbox');
```

Comment:
This should be simplified to `const lightboxGroupId = element.getAttribute('lightbox') || ...`

Reply:
Does this still read well? 

```
      const lightboxGroupId = element.getAttribute('lightbox') ||
       'carousel' + (element.getAttribute('id') || this.counter_++);
```

Reply:
imho it's more readable since the `const` and `||` communicates a fallback value better than a `let` and an `if` statement. But if you feel like it's more cluttered this way, follow your own style-compass : )